collect_app/src/main/java/org/odk/collect/android/material/MaterialFullScreenDialogFragment.java

```diff
@@ -39,8 +39,10 @@ public void onStart() {
             int height = ViewGroup.LayoutParams.MATCH_PARENT;
             dialog.getWindow().setLayout(width, height);
 
-            // Make sure soft keyboard shows for focused field - annoyingly needed
-            dialog.getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_VISIBLE);
+            if (shouldShowSoftKeyboard()) {
```

Comment:
Oh wow was this making it pop open the keyboard even though there aren't any fields? That's annoying. Feel free to reverse it so the default is `false` and the other dialogs override.

Reply:
Fixed.